x-pack/plugins/observability_solution/inventory/server/routes/entities/route.ts

```diff
@@ -58,15 +63,40 @@ export const listLatestEntitiesRoute = createInventoryServerRoute({
 
     const { sortDirection, sortField, entityTypes, kuery } = params.query;
 
-    const latestEntities = await getLatestEntities({
-      inventoryEsClient,
-      sortDirection,
-      sortField,
-      entityTypes,
+    const [alertsClient, latestEntities] = await Promise.all([
+      createAlertsClient({ plugins, request }),
+      getLatestEntities({
+        inventoryEsClient,
+        sortDirection,
+        sortField,
+        entityTypes,
+        kuery,
+      }),
+    ]);
+
+    const identityFieldsPerEntityType = getIdentityFieldsPerEntityType(latestEntities);
+
+    const alerts = await getLatestEntitiesAlerts({
+      identityFieldsPerEntityType,
+      alertsClient,
```

Comment:
Another way of doing it is by quering the entity definitions itself, something like:
```
const entityManagerStartClient = await plugins.entityManager.start();
    const definitions = await (
      await entityManagerStartClient.getScopedClient({ request })
    ).getEntityDefinitions({ builtIn: true });
```

This way you can fetch both the **Latest Entites** and **Latest Entities alerts** in parallel.